docs/bot-usage.md

```diff
@@ -56,8 +56,8 @@ freqtrade -c path/far/far/away/config.json
 
 The bot allows you to use multiple configuration files by specifying multiple
 `-c/--config` configuration options in the command line. Configuration parameters
-defined in the last configuration file override parameters with the same name
-defined in the previous configuration file specified in the command line.
+defined in latter configuration files override parameters with the same name
+defined in the earlier configuration files specified in the command line.
```

Comment:
"...defined in the previous configuration files specified in the command line earlier." (?)